packages/contracts/src/forwarder/IForwarder.sol

```diff
@@ -4,8 +4,22 @@ pragma abicoder v2;
 
 import "@openzeppelin/contracts/interfaces/IERC165.sol";
 
+/**
+ * @title The Forwarder Interface
+ * @notice The contracts implementing this interface take a role of authorization, authentication and replay protection
+ * for contracts that choose to trust a `Forwarder`, instead of relying on a mechanism built into the Ethereum protocol.
+ *
+ * @notice if the `Forwarder` contract decides that an incoming `ForwardRequest` is valid, it must append 20 bytes that
+ * represent the caller to the `data` field of the request and send this new data to the target address (the `to` field)
+ *
+ * :warning: **Warning** :warning: The Forwarder can have a full control over a `Recipient` contract.
```

Comment:
Recipient contracts should only trust forwarders that passed through security audit, otherwise they are susceptible to identity theft.

Reply:
Fixed.